avm/res/app-configuration/configuration-store/key-value/main.bicep

```
@@ -0,0 +1,40 @@
+metadata name = 'App Configuration Stores Key Values'
+metadata description = 'This module deploys an App Configuration Store Key Value.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. Name of the key.')
+param name string
+
+@description('Required. Name of the value.')
```

```suggestion
@description('Required. The value of the key-value.')
```
Isn't it rather this?